### PR TITLE
add pesign

### PR DESCRIPTION
**Pull request by ddstreetmicrosoft**
add pesign package

### CODE REVIEW

**Comment by ddstreetmicrosoft** [comment]
buddybuild https://dev.azure.com/mariner-org/mariner/_build/results?buildId=548510&view=results

**Review by Camelron — APPROVED** [review]
lgtm

File: SPECS/pesign/0001-cms_common-Fixed-Segmentation-fault.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nicolas Frayer <nfrayer@redhat.com>
+Date: Mon, 20 Feb 2023 15:26:20 +0100
+Subject: [PATCH] cms_common: Fixed Segmentation fault
+
+When running efikeygen, the binary crashes with a segfault due
```
Comment (Camelron) [inline SPECS/pesign/0001-cms_common-Fixed-Segmentation-fault.patch:6]: How did a fix like this not make it into the 116 release?

Comment (ddstreetmicrosoft) [reply SPECS/pesign/0001-cms_common-Fixed-Segmentation-fault.patch:6]: yeah that's a great question :)

**Pull request merged**